moveit_ros/planning_interface/planning_scene_interface/src/planning_scene_interface.cpp

```diff
@@ -312,16 +347,65 @@ class PlanningSceneInterface::PlanningSceneInterfaceImpl
     }
   }
 
+  void connectGetPlanningSceneService()
+  {
+    if (persistent_connections_)
+    {
+      ros::service::waitForService(move_group::GET_PLANNING_SCENE_SERVICE_NAME);
+    }
+
+    planning_scene_service_ = node_handle_.serviceClient<moveit_msgs::GetPlanningScene>(
+        move_group::GET_PLANNING_SCENE_SERVICE_NAME, persistent_connections_);
+
+    if (wait_)
+    {
+      waitForService(planning_scene_service_);
+    }
+    else
+    {
+      if (!planning_scene_service_.exists())
+      {
+        throw std::runtime_error("ROS service not available");
```

Comment:
Throwing an exception seems not to be a good idea. Nobody catches the exception.

Reply:
I only moved what was already implemented in this [line](https://github.com/moveit/moveit/blob/9caa14eabc69c6ba2328eacfb5f6253a8acff085/moveit_ros/planning_interface/planning_scene_interface/src/planning_scene_interface.cpp#L72) in the master branch. What should I do? Should I just replace it with a return false?

Reply:
I see. Then keep it as is.